orchagent/p4orch/acl_table_manager.cpp

```diff
@@ -36,16 +36,15 @@ AclTableManager::AclTableManager(P4OidMapper *p4oidMapper, ResponsePublisherInte
     SWSS_LOG_ENTER();
 
     assert(p4oidMapper != nullptr);
-    // Create the default UDF match
-    auto status = createDefaultUdfMatch();
```

Comment:
Where is it created now?

Reply:
Line#468, when the ACL table with UDF fields defined created for the first time by p4orch.